src/plot_api/subroutines.js

```diff
@@ -43,7 +43,9 @@ function overlappingDomain(xDomain, yDomain, domains) {
 exports.lsInner = function(gd) {
     var fullLayout = gd._fullLayout;
     var gs = fullLayout._size;
+    var pad = gs.p;
     var axList = Plotly.Axes.list(gd);
+    var hasSVGCartesian = fullLayout._has('cartesian');
```

Comment:
Ah right - I figured out that that's the reason, which is why I added `SVG` to the var name, but perhaps I should add the full explanation in a comment. Will become important when we start using SVG axes with GL2D plots @dfcreative